fundamentals/intro.tex

```diff
@@ -83,19 +84,20 @@ \subsection{Background}
 
 Nuclear fuel cycle simulators drive \gls{RDD} by calculating `metrics',
 quantitative measures of performance that can be compared among fuel cycle
-options. The feasibility of the technology development and deployment
-strategies which comprise a fuel cycle option, the dynamics of transitions
-between fuel cycles, and many other measures of performance can be expressed in
-terms of these metrics. For example, economic feasibility is often measured in
-\gls{LCOE}, requiring calculation of lifetime
-fuel costs and electricity generation, while environmental performance might be
-measured by spent fuel volume, toxicity, or mined uranium.  A meta-analysis of
-fuel cycle systems studies identified over two dozen unique quantitative
-metrics spanning economics and cost, environmental sustainability and waste
-management impacts, safety, security and nonproliferation, resource adequacy
-and utilization, among others. With few exceptions, these metrics are derived
-from mass balances calculated by a fuel cycle simulator.
-\cite{flicker_evaluation_2014}.
+options.  The feasibility of the technology development and deployment
+strategies which comprise a fuel cycle option, the operational features of
+nuclear energy systems, the dynamics of transitions between fuel cycles, and
+many other measures of performance can be expressed in terms of these metrics.
+For example, economic feasibility is often measured in \gls{LCOE}, requiring
+calculation of lifetime fuel costs and electricity generation, while
+environmental performance might be measured by spent fuel volume, toxicity, or
+mined uranium.  A meta-analysis of fuel cycle systems studies identified over
+two dozen unique quantitative metrics spanning economics and cost,
+environmental sustainability and waste management impacts, safety, security and
+nonproliferation, resource adequacy and utilization, among others
+\cite{flicker_evaluation_2014}. With few exceptions (e.g.
+\cite{poinssot_assessment_2014}), these metrics are derived from mass balances
```

Comment:
Yes I agree. It sounds like "It would be nice to illustrate what are the
few exceptions" is asking explicitly for a few examples.

On Tue, May 12, 2015 at 12:33 PM, Katy Huff <notifications@github.com>
wrote:

> In fundamentals/intro.tex
> <https://github.com/cyclus/fundamentals-paper/pull/131#discussion_r30158864>
> :
>
> > -and utilization, among others. With few exceptions, these metrics are derived
> > -from mass balances calculated by a fuel cycle simulator.
> > -\cite{flicker_evaluation_2014}.
> > +options.  The feasibility of the technology development and deployment
> > +strategies which comprise a fuel cycle option, the operational features of
> > +nuclear energy systems, the dynamics of transitions between fuel cycles, and
> > +many other measures of performance can be expressed in terms of these metrics.
> > +For example, economic feasibility is often measured in \gls{LCOE}, requiring
> > +calculation of lifetime fuel costs and electricity generation, while
> > +environmental performance might be measured by spent fuel volume, toxicity, or
> > +mined uranium.  A meta-analysis of fuel cycle systems studies identified over
> > +two dozen unique quantitative metrics spanning economics and cost,
> > +environmental sustainability and waste management impacts, safety, security and
> > +nonproliferation, resource adequacy and utilization, among others
> > +\cite{flicker_evaluation_2014}. With few exceptions (e.g.
> > +\cite{poinssot_assessment_2014}), these metrics are derived from mass balances
>
> I am ok with "(e.g. water pollution and land use
> \cite{poinssot_assessment_2014})" but I think listing all of the ones in
> that paper that are not mass flows is a bit out of scope and give too much
> attention to this side-note.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/fundamentals-paper/pull/131/files#r30158864>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192